CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## 0.52.0 / 2021-11-03
+
+* [CHANGE] Automatically add sharding `relabel_configs` to all scrape jobs in `additionalScrapeConfigs` if `shards > 1`. #4324
+* [CHANGE] Remov app label from prometheus, alertmanager and thanosruler statefulsets/pods. #4350
+* [FEATURE] Add alertRelabelConfigs field to the Thanos Ruler CRD for configuring Prometheus alert relabeling features. #4303
+* [FEATURE] Add support for updated matching syntax in Alertmanager's raw config for `inhibit_rules`. #4307
+* [FEATURE] Add support for updated matching syntax in Alertmanager's raw config for `route`. #4309
+* [FEATURE] Add "generic ephemeral storage" as a data storage option for Alertmanager, Prometheus and Thanos Ruler. #4326
+* [FEATURE] Add validating webhook for AlertManagerConfig. #4338
+* [FEATURE] Add input validations for duration fields in alertmanager and thanos. #4352
+* [EHNAHCEMENT] Improve docs and error message for "smarthost" field. #4299
+* [ENHANCEMENT] Add alerts for config reloader sidecars. #4294
```

Comment:
I am on the fence about adding this. On one hand, alerts are essentially docs and docs shouldn't be in the changelog. On the other hand it might be worth mentioning that we have alerts :thinking:

Reply:
I think it's worth communicating that alerts were added so that people that run Prometheus can use them.